edweiss-app/components/courses/AssignmentDisplay.tsx

```diff
@@ -79,11 +83,7 @@ const AssignmentDisplay: ReactComponent<{ item: AssignmentWithColor, index: numb
 
     return (
         isSwipeable ?
-            <Swipeable
-                ref={(ref) => { swipeableRefs.current[index] = ref; }} renderRightActions={renderRightActions} onSwipeableOpen={(direction) => {
-                    if (direction === 'right') { console.log(`Swipe detected on assignment: ${item.name}`); saveTodo(item.name, item.dueDate, item.type); swipeableRefs.current[index]?.close(); }
-                }
-                }>
+            <Swipeable testID={testIDs.swipeableComponent} ref={(ref) => { swipeableRefs.current[index] = ref; }} renderRightActions={renderRightActions} onSwipeableOpen={(direction) => { if (direction === 'right') { console.log(`Swipe detected on assignment: ${item.name}`); saveTodo(item.name, item.dueDate, item.type); Toast.show({ type: 'success', text1: t(`course:toast_added_to_todo_text1`), text2: t(`course:toast_added_to_todo_text2`), }); swipeableRefs.current[index]?.close(); } }}>
```

Comment:
Same here, jamming it all in one line makes refactoring and working with it very cumbersome.